src/foam/nanos/auth/Credentials.js

```diff
@@ -0,0 +1,25 @@
+foam.CLASS({
+    package: 'foam.nanos.auth',
+    name: 'UsernamePasswordCredentials',
+    properties: [
+        {
+            class: 'String',
+            name: 'username'
+        },
+        {
+            class: 'String',
+            name: 'password'
+        }
+    ]
+});
+
+foam.CLASS({
```

Comment:
We discourage having multiple models per file now, since doing so interferes with our new staging feature.